packages/runtime/src/next/document.tsx

```diff
@@ -33,13 +30,9 @@ export class Document extends NextDocument<DocumentProps> {
   }
 
   render() {
-    const { isPreview } = this.props.__NEXT_DATA__
-
     return (
       <Html>
-        <Head>
-          <PreviewModeScript isPreview={isPreview} appOrigin={this.props.appOrigin} />
-        </Head>
+        <Head></Head>
```

Comment:
Can we remove this line? If not, let's change to a self-closing tag. i.e., `<Head />`